joss_paper/paper.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Zarrtraj: A Python package for streaming molecular dynamics trajectories from cloud services'
+tags:
+  - streaming
+  - molecular-dynamics
+  - file-format
+  - mdanalysis
+  - zarr
+authors:
+  - name: Lawson Woods
+    orcid: 0009-0003-0713-4167
+    affiliation: 1 
+  - name: Hugo Macdermott-Opeskin
+    orcid: 0000-0002-7393-7457
+    affiliation: 1
+  - name: Edis Jakupovic 
+    affiliation: 1
+  - name: Yuxuan Zhuang
+    orcid: 0000-0003-4390-8556
+    affiliations: 1
+  - name: Richard J Gowers
+    orcid: 0000-0002-3241-1846
+    affiliations: 1
+  - name: Oliver Beckstein
+    orcid: 000-0003-1340-0831
+    affiliation: 1
+affiliations:
+ - name: Placeholder
+   index: 1
+date: 22 September 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Molecular dynamics (MD) simulations provide a microscope into the behavior of 
+atomic-scale environments otherwise prohibitively difficult to observe, however,
+the resulting trajectory data is too often siloed in a single institutions' 
+HPC environment, rendering it unusable by the broader scientific community.
+Zarrtraj enables these trajectories to be read directly from cloud storage providers
+like AWS, Google Cloud, and Microsoft Azure into MDAnalysis, a popular Python 
+package for analyzing trajectory data, providing a method to open up access to
+trajectory data to anyone with an internet connection. Enabling cloud streaming
+for MD trajectories empowers easier replication of published analysis results,
+analyses of large, conglomerate datasets from different sources, and training
+machine learning models without downloading and storing trajectory data.
+
+# Statement of need
+
+The computing power in HPC environments has increased to the point where
+running simulation algorithms is often no longer the constraint in
+obtaining scientific insights from molecular dynamics trajectory data. 
+Instead, the ability to process, analyze and share large volumes of data provide 
+new constraints on research in this field.
+
+Other groups in the field recognize this same need for adherence to 
```

Comment:
I think MDsrv https://academic.oup.com/nar/article/50/W1/W483/6593534 should also be mentioned here.

Reply:
other papers worth citing:

- Bringing Molecular Dynamics Simulation Data into View https://www.cell.com/trends/biochemical-sciences/fulltext/S0968-0004(19)30137-9
- GPCRmd uncovers the dynamics of the 3D-GPCRome
 https://www.nature.com/articles/s41592-020-0884-y
- Sharing Data from Molecular Simulations https://pubs.acs.org/doi/full/10.1021/acs.jcim.9b00665?casa_token=Ivo26Sn_xFAAAAAA%3AwJLKekUvaYfOPZyi97LPZR7zir_x9cWlcbEb8UOZN7ZTqeNpmPE96rsLChkot3xPeGUtR4Es42lx7VKFyg

Reply:
emphasizing that they only provide predefined analysis results or simple geometric features.